spec/models/booking_response_spec.rb

```diff
@@ -57,6 +57,18 @@
         end
       end
 
+      context 'and the allowed visitor is a child' do
+        let(:visit) { create(:visit) }
+
+        before do
+          visit.visitors.update_all(date_of_birth: 5.years.ago)
```

Comment:
Not sure if its `adult_age` aware, I'll double check, I'll add spec for that to make it more explicit.